aya-obj/src/obj.rs

```diff
@@ -47,17 +48,24 @@ pub struct Features {
     bpf_perf_link: bool,
     bpf_global_data: bool,
     bpf_cookie: bool,
+    cpumap_prog_id: bool,
```

Comment:
Were these all added at different times? We can't probe once?

Reply:
sadly no:
- cpumap prog id: 5.9
- devmap prog id: 5.8
- devmap hash: I think 5.8 too
merging devmap and devmap hash

Reply:
alright :(